### PR TITLE
UG: Update find, edit, help commands

### LINKED ISSUE #172: [PE-D][Tester D] Edit command does not show that at least 1 field is required in the UG
-------------
Labels: `severity.Low` `type.DocumentationBug`
original: TheSoggy/ped#6

### LINKED ISSUE #188: [PE-D][Tester E] Inappropriate use of 'person' in edit command for User Guide.
In the User Guide, 'person' is used without specifying, which can make it confusing for the reader. It should be more consistent by using 'employee'.

Screenshot:

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: kangzongxian/ped#13

### PR DESCRIPTION
**Pull request by kevinchangjk**
Sorry branch name is wrong.

Primarily for #157, for my main part.
Also resolves #172 and fixes #188 in particular.

Additional notes:

- Have added a new section for the `help` command
    - Still lacking a screenshot as the help window itself is not updated
- Have added a new section on what the user can expect to see in an employee profile
    - Includes important disclaimers on the use of Employee ID (cannot be manipulated by them)
    - Does not include details on department, leave, rate (simply links to relevant sections)
- Not sure if hyperlinks are working honestly, and also not sure how to test before merging

### CODE REVIEW

**Comment by ngshijun** [comment]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -240,21 +304,42 @@ Format: `list`
 
 #### Finding an employee : `find`
 
-Finds employees by searching for specified keywords. You can search for a specific name, position, or department, or with any combination of the aforementioned.
+Once you get familiar with Coydir and batch-adding numerous employees into the database, using `list` simply isn't sufficient to look for the right employee.
```
Comment (prit3010) [inline docs/UserGuide.md:313]: I like the personal touch here!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -240,21 +304,42 @@ Format: `list`
...
+    <li>The keywords you input for searching are <strong>case-insensitive</strong>. Additionally, they need not be full words, just a segment will do. This means that searching for the department "tech" will show results for both "Technology and "Information Technology".</li>
+  </ul>
+</div>
+
+![Find Command - Example](./images/ui-screenshots/find-example.png)
+
 Example:
 
-- `find n/John j/engineer d/Tech` displays the employee "John Doe", who is a "Software Engineer" in the "Information Technology" department.
+- `find j/officer d/Management` displays 2 employees, who are "Officers" in the "General Management" department.
```
Comment (prit3010) [inline docs/UserGuide.md:344]: Hmm for this part maybe can give more examples?

I was thinking of :
1 example of only one parameter
1 example of 2 parameters 
1 example of 3 parameters

**Review by prit3010 — CHANGES_REQUESTED** [review]
Perhaps can add a few more examples for find!

File: docs/UserGuide.md (hunk 2 above)
Comment (kevinchangjk) [reply docs/UserGuide.md:344]: Yeah can add one for 3 parameters (to demonstrate usage for highly specific searches), but I think another example for 1 parameter is not necessary. 

The purpose of the 2 parameter example is firstly to show the use of the command, secondly to show the stacking of keyword matching, and lastly, to show that it is capable of returning all employees who fit the criteria. To that extent I think the example for 1 parameter is not needed here.

File: docs/UserGuide.md (hunk 1 above)
Comment (chao890) [reply docs/UserGuide.md:313]: I also like your personal touch

**Review by chao890 — APPROVED** [review]
LGTM

**Review by prit3010 — APPROVED** [review]
LTGM

**Pull request merged**